src/filter/proxy.rs

```diff
@@ -232,8 +232,12 @@ pub unsafe extern "system" fn notify_rename<T: SyncFilter + 'static>(
     if let Some(filter) = filter_from_info::<T>(info) {
         let request = Request::new(*info);
         let ticket = ticket::Rename::new(request.connection_key(), request.transfer_key());
+        let info = info::Rename(
+            (*params).Anonymous.Rename,
+            request.volume_letter().to_os_string(),
+        );
```

Comment:
Ah, this is so ugly but it must be done.

Reply:
Cause I do the same thing in the Placeholder::path